inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/AnnotationQueueOverviewDataProvider.java

```diff
@@ -94,50 +116,59 @@ public TableContentProvider(
         newList.sort((o1, o2) ->
         {
             int dir = getSort().isAscending() ? 1 : -1;
-            if (getSort().getProperty().equals(headers.get(0))) {
+            if (getSort().getProperty().equals(headers.get(0)))
+            {
                 return dir * (o1.getName().compareTo(o2.getName()));
             }
-            else if (getSort().getProperty().equals(headers.get(1))) {
+            else if (getSort().getProperty().equals(headers.get(1)))
+            {
                 return dir * Long.compare(getFinishedAmountForDocument(o1),
                         getFinishedAmountForDocument(o2));
             }
-            else if (getSort().getProperty().equals(headers.get(2))) {
+            else if (getSort().getProperty().equals(headers.get(2)))
+            {
                 return dir * Long.compare(getInProgressAmountForDocument(o1),
                         getInProgressAmountForDocument(o2));
             }
-            else if (getSort().getProperty().equals(headers.get(3))) {
+            else if (getSort().getProperty().equals(headers.get(3)))
+            {
                 return dir * (Integer.compare(getUsersWorkingOnTheDocument(o1).length(),
                         getUsersWorkingOnTheDocument(o2).length()));
 
             }
-            else if (getSort().getProperty().equals(headers.get(4))) {
-                if (o1.getUpdated() == null) {
+            else if (getSort().getProperty().equals(headers.get(4)))
+            {
+                if (o1.getUpdated() == null)
+                {
                     return dir;
                 }
-                else if (o2.getUpdated() == null) {
+                else if (o2.getUpdated() == null)
+                {
                     return dir * -1;
                 }
-                else {
+                else
+                {
                     return dir * (o1.getUpdated().compareTo(o2.getUpdated()));
                 }
             }
-            else {
+            else
+            {
                 return 0;
             }
         });
 
 
         //Reset
-        this.shownDocuments.clear();
+        this.shownDocuments = new ArrayList<>();
         shownDocuments.addAll(newList);
 
 
-        if ((int)aFirst + (int)aCount > newList.size())
+        if ((int)first + (int)count > newList.size())
```

Comment:
Why cast? Why not work with ints in the first place?

Reply:
Due to @Override from superclass, long must be used